### PR TITLE
Create project details popup

### PR DESCRIPTION
**Pull request by emmiiorji**
@Fatima-hub333 and I have implemented the popup window for displaying details of the projects.

It implements the following features on both mobile and desktop:

- [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.

Find screenshots below:
image
image

Thank you.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -16,8 +20,153 @@ function toggleOpenMenu() {
...
+
+const markupModal = (projectId) => {
+  const project = projects[projectId];
+  return `<div class="project-margin">
+            <div class="project-modal-container">
+              <div class="modal-header">
+                <h2>${project.name}</h2>
+                <img src="./assets/img/close-icon.png" class="close" alt="Close Modal">
+              </div>
+              ${createLanguagesUlElement(project.languages)}
```
Comment (Whoistolu) [inline js/script.js:40]: - [ ] [REQUIRED] You have indeed done a great job @emmiiorji , your popup follows the design for the mobile version from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471), however, the popup for the desktop language buttons needs a resize. The current language button on your design looks too big which does not make the UI look professional. Kindly reduce the size and ensure your design looks more like the figma template popup.
### Your Design
Screenshot from 2022-04-14 19-44-22
### Figma Template 2
Screenshot from 2022-04-14 19-44-39

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @emmiiorji 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/script.js (hunk 1 above)
Comment (Donard97) [inline js/script.js:40]: - [ ] Kindly do the changes that the other reviewer requested you to do. You have to decrease the size of the list of languages following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Keep coding 💪 

Here you have a screenshot from your project:

image

Here you have a screenshot from the design:

image

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi @emmiiorji , 👋  

_You did an amazing job so far! 👏 💯_ 
    
**You are already there but you need one more step to pass.  💪** 

133678845-1b015b27-e9bc-4784-a316-bc699db8237a

## Highlights ✅ 

✔️ There is no linter error
✔️ Code is clean
✔️ PR is descriptive
✔️ Readme file is professional

## Required Changes ♻️

- [ ] Please check the comments under the review.
    
## Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
    
**Cheers and Happy coding!👏👏👏**
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
    
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
    
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
    ```

File: js/script.js (hunk 1 above)
Comment (emmiiorji) [reply js/script.js:40]: Hi @Donard97 

Thank you for your kind feedback. The change has already been effected. Kindly find in the image below:
image


Sorry, I didn't note that the push was unsuccessful before requesting another review. 

Thanks

**Review by smunozmo — APPROVED** [review]
Hi @emmiiorji, 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**